docs/howto_guides/api_gui.md

```diff
@@ -0,0 +1,161 @@
+# Using the NMDC API Graphical User Interface (GUI)
+
+## Retrieving and Submitting Metadata using the ___Find___ and ___Metadata___ API Endpoints
+
+Metadata describing NMDC data (e.g. studies, biosamples, data objects, etc.) may be retrieved or submitted with GET and POST requests, respectively, using the **[NMDC API Graphical User Interface (GUI)](https://api.microbiomedata.org/docs#/)**. The API GUI provides a user interface for programmatic access to the NMDC data portal without needing to use the Command Line.
```

Comment:
retrieved or submitted... do we need submitted for a user documentation?

Reply:
Removed all the metadata "submission" and post topics. See commit [c96e9ee](https://github.com/microbiomedata/NMDC_documentation/pull/27/commits/c96e9ee6fa5dc0e3579412152b1e103909a2a897)